src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
+          <h4>
+            <input type="checkbox" [checked]="playPlaylist" (change)="togglePlayPlaylist()">
+            Or: Make It A Playlist
```

Comment:
Yeah, I agree that it should only show two iframe options. Right now preview always shows the wide version of the player -- with or without playlist depending on whether it's selected. 
So -- shall I try putting "make it a playlist" above the "customize your player" form, and we'll see if that feels more intuitive?